tanner/reporting/log_local.py

```diff
@@ -1,15 +1,23 @@
-import os
 import json
+import os
 
 from tanner import config
 
+
 class Reporting():
     def __init__(self):
-        # check if file exists, else create
-        pass
+        self.report_file = config.TannerConfig.get('LOCALLOG', 'PATH')
+        if not os.path.exists(self.report_file):
+            with open(self.report_file, 'w') as out:
+                json.dump([], out)
 
     def create_session(self, session_data):
-        report_file = config.TannerConfig.get('LOCALLOG', 'PATH')
-        with open(report_file, 'a') as out:
-            out.write('{0}\n'.format(session_data))
-        return '1'
+        with open(self.report_file, 'br+') as out:
+            out.seek(-1, 2)
+            out.truncate()
+
+        with open(self.report_file, 'a') as out:
+            if os.path.getsize(self.report_file) != 1:
+                out.write(",")
+            json.dump(session_data, out)
+            out.write("]")
```

Comment:
If you like with timestamp:
Use this code instead of `out.write("]")`
```
 with open(self.report_file, 'br+') as out:
            out.seek(-1,2)
            out.truncate()
        with open(self.report_file, 'a') as out:
            out.write(", \"timestamp\": \""+ datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f') + "\"}]")
```